### ISSUE TITLE
Ctrl+W L does not switch from Side Bar to the currenly displayed editor

### ISSUE THREAD

**Issue by dmitriibundin** [VSCodeVim/Vim#7020]
**Describe the bug**
When Side Bar  (e.g. Explorer) is focused pressing Ctrl+W L does not switch to the currently displayed editor
 
**To Reproduce**
1. Make the Side Active
2. Turn Ctrl+W L

**Expected behavior**
Expected that currently displayed editor is activated and ready for intput

**Screenshots**
![Screenshot from 2021-08-22 12-32-45](https://user-images.githubusercontent.com/55092833/130350184-a8a151a9-ba33-4ab2-b02e-d7c4a1a7257b.png)

**Environment (please complete the following information):**

- Extension (VsCodeVim) version: 1.21.6
- VSCode version: 1.59.0
- OS: Ubuntu 20.04

**Additional context**
Any default binding related to Ctrl+W which closes the currenly displayed editor and L which opens the given file are removed as follows

**Comment by TryerGit** [comment]
As a user, I find VSCode's default access of sidebar via `Ctrl+0` and the next editor to the right via `Ctrl+1` rather intuitive. Closing the current open editor via `Ctrl+w` also seems okay for me.

Vim's version of this has an extra keystroke, first `Ctrl+w` then h or j or k or l.

**Comment by dmitriibundin** [comment]
@TryerGit Agree, Ctrl+1/2/etc works fine, but I find it counter-intuitive to be able to switch between editor's windows with the usual Vim shortcuts Ctrl+L/Ctrl+H/Ctrl+J as well as to switch into the Side bar and terminal panel. But it does not work vice versa. 

I think Vim users might get used to usual vim way to switch between the side bar and the editor windows (with plugins like nerdtree, etc...)

**Comment by goodshort** [comment]
Workaround to get the usual vim behaviour of `<C-w>l` to switch back to editor:
1. Open the [keybindings.json](https://github.com/VSCodeVim/Vim#remapping-more-complex-key-combinations) to create a complex key combinations
2. Add the following:
```json
{
	"key": "ctrl+w l",
	"command": "workbench.action.focusFirstEditorGroup",
	"when": "sideBarFocus"
}
```

_Originally posted by @mogelbrod in https://github.com/VSCodeVim/Vim/issues/1988#issuecomment-1304600984_
